releasenotes/notes/add_snmp_symbol_forced_type-ffc92909484115c4.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    [corechecks/snmp] Add symbol level ``forced_type`` for table metrics.
```

Comment:
🥜 : We should decide either on `forced_type` or `metric_type`. My gut reaction is to stick with `foced_type` since that exists at the root level already rather than introducing a new term.

Reply:
@ken-schneider Yeah, I thought about it, but `forced_type` really seems not a good name, it doesn't tell what it's "forcing".

Here we are adding the metric type to the `symbol` scope, and it really doesn't feel right to add a `forced_type` at this level.

Maybe we can rename the root level `forced_type` to `metric_type` and then deprecate `forced_type`. What do you think?